observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,172 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-forwarding-logs"]
+= Forwarding Logs to a LokiStack instance
+include::_attributes/common-attributes.adoc[]
+:context: otel-forwarding-logs
+
+To configure forwarding logs to a LokiStack instance, you can deploy and configure the OpenTelemetry Collector. You can deploy the OpenTelemetry Collector in the deployment mode by using the specified processors, receivers, and exporters.
+
+.Prerequisites
+
+* The {OTELOperator} is installed.
+* The {loki-op} is installed. See _Additional resources_ on how to install LokiStack.
+* A supported LokiStack instance is deployed on the cluster.
+
+TODO add dev-preview note and that the `lokipush` exporter will be removed in future release in favour of OTLPHTTP.
```

Comment:
@pavolloffay, is this a dev preview or tech preview?

Reply:
This note should be rephrased.

The `lokipush` exporter will be changed to OTLP HTTP in the future release when the OpenShift LokiStack adds supports for OTLP HTTP.